modules/graphdb/nsg.tf

```diff
@@ -51,8 +54,8 @@ resource "aws_security_group_rule" "graphdb_network_lb_ingress" {
   description       = "CIRDs allowed to access GraphDB."
   security_group_id = aws_security_group.graphdb_security_group.id
   type              = "ingress"
-  from_port         = 7200
-  to_port           = 7200
+  from_port         = var.graphdb_node_count == 1 ? 7201 : 7200
```

Comment:
Could calculate them once in some `local {}`

Reply:
done